CONTRIBUTING.md

```diff
@@ -2,7 +2,13 @@
 
 ## Generate Protobuf Go Files
 
-Make sure you have Docker installed.
+1. Make sure `opamp-go/internal/opamp-spec` submodule files exist. You can do this
+either by cloning the `opamp-go` repo with submodules the first time, e.g. 
+`git clone --recurse-submodules git@github.com:open-telemetry/opamp-go.git`.
+Alternatively, if you already cloned `opamp-go` without the submodule then
+execute `git submodule update --init` to fetch all files for `opamp-spec` submodule.
 
-Run `make gen-proto`. This should compile `internal/proto/*.proto` files to 
+2. Make sure you have Docker installed.
```

Comment:
Technically correct, you need to have it installed … and running! ;)

Reply:
Added.